cmd/kogito/command/shared/resource_checks.go

```diff
@@ -67,7 +67,7 @@ func CheckProjectExists(kubeCli *client.Client, namespace string) error {
 	return nil
 }
 
-func checkKogitoOperatorExists(kubeCli *client.Client, namespace string) (bool, error) {
+func CheckKogitoOperatorExists(kubeCli *client.Client, namespace string) (bool, error) {
```

Comment:
This function should moved to `pkg/infrastructure`

Reply:
operator go file created into pkg/infrastructure